cmd/juju/space/space.go

```diff
@@ -137,6 +140,10 @@ func (m *mvpAPIShim) ListSpaces() ([]params.Space, error) {
 	return m.facade.ListSpaces()
 }
 
+func (m *mvpAPIShim) ReloadSpaces() error {
+	return m.facade.ReloadSpaces()
```

Comment:
seems strange to need an "mvpAPIShim". I wonder if there is something we should flag here to come back and get rid of.
'mvp' is short for "minimum viable product" which is usually "quickest and dirtiest thing to get something out the door".